CHANGES.txt

```diff
@@ -5,6 +5,13 @@ Changes for crate
 Unreleased
 ==========
 
+2017/05/17 0.19.3
+=================
+
+ - Fix bulk updates which were broken due to query rewrites.
+ - Fix typo to DB API in README
```

Comment:
There's no need to mention the README fix - only end-user relevant changes should end up in the changes.txt